.github/workflows/tests.yml

```diff
@@ -0,0 +1,202 @@
+name: Tests
+on:
+  push:
+    branches: [master]
```

Comment:
Since we are moving the repository, is it an opportunity to modernize the default branch name to `main` in the same breath?

Reply:
I'm +1 on switching to `main`, but can we do it as a separate PR after finishing this PR and #2? Otherwise we'll have to reopen them against `main` instead and we'll lose current reviews/comments.